app/src/main/java/es/wolfi/app/passman/CredentialEdit.java

```diff
@@ -329,14 +412,30 @@ public void onClick(View view) {
             public void onSuccess(int statusCode, cz.msebera.android.httpclient.Header[] headers, byte[] responseBody) {
                 String result = new String(responseBody);
                 if (statusCode == 200 && !result.equals("")) {
-                    Snackbar.make(view, R.string.successfully_updated, Snackbar.LENGTH_LONG)
-                            .setAction("Action", null).show();
-                    assert getFragmentManager() != null;
+                    try {
```

Comment:
This code seems to be repeated a few times, wonuldn't it be better if it was moved to it's own function? 
I'm not sure if it might make sense to move it to the api library or if it's too specific for that.

Reply:
what do you mean exactly?
I think its better to define the responseHandlers where they are now. They are all a bit (some more, some less) different and need custom variables from the "outer scope".